hack/tools/release/notes/print.go

```diff
@@ -109,10 +109,10 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 	fmt.Printf("## Changes since %s\n", p.fromTag)
 
 	fmt.Printf("## :chart_with_upwards_trend: Overview\n")
-	if count := len(entries); count == 1 {
+	if commitCount == 1 {
 		fmt.Println("- 1 new commit merged")
-	} else if count > 1 {
-		fmt.Printf("- %d new commits merged\n", count)
+	} else if commitCount > 1 {
+		fmt.Printf("- %d new commits merged\n", commitCount)
```

Comment:
I'm not sure we need this
Is the intent here to always show the exact number of new commits contained in a release regardless of the amount of entries printed? In that case nit: I would suggest a different name, like `commitsInRelease`